### DISCUSSION TITLE
Disable tab default behavior

### DISCUSSION THREAD

**Discussion by bohmax** [vueup/vue-quill#295] [Ideas]
Hello,
I want to disable the tab default behavior of quill. I wish to go in the next item in the dom instead of tabbing in the editor.
With Quill I can to this with
```
modules: {
    keyboard: {
      bindings: {
        tab: {
          key: 9,
          handler: function () {
            console.log('lol')
          }
        }
      }
    }
  }
```
  But i couldn't achieve the same behavior in VueQuill

**Comment by bohmax** [comment]
In the end I did
```
onMounted(() => {
  const bindings = myEditor.value.getQuill().getModule('keyboard').bindings
  delete bindings['9']
  console.log(bindings)
})
```
It works, but I think it should be possible to editing the keybindings in the options props

**Comment by iPrytz** [comment]
I just did the same, but had a slightly different approach: 
at event: @ready="onEditorReady($event)
```
function onEditorReady(quill) {
   delete quill.keyboard.bindings[9]
   // And this to remove tabindex from toolbar to directly jump to next input element
   const toolButtons = quill.container.parentElement.querySelectorAll('.ql-toolbar button')
   toolButtons.forEach((item) => { item.setAttribute('tabindex', '-1'); })
}
```
but you can also override keybindings in globalOptions, that did'nt solve the tab outside problem directly though:

```
const globalOptions = {
  modules: {
    keyboard: {
      bindings: {
        tab: {
          key: 'Tab',
          handler() {
            this.quill.blur()
            return false;
          },
        },
      }
    }
  },
}
```
`QuillEditor.props.globalOptions.default = () => globalOptions`
